modules/managed-cluster-notification-troubleshooting.adoc

```diff
@@ -0,0 +1,20 @@
+[id="troubleshoot_{context}"]
+= Troubleshooting
+
+.If you are not receiving cluster notification emails
+* Ensure that email from sent from `@redhat.com` addresses is not filtered out of your email inbox.
+* Ensure that your correct email address is listed as a notification contact for the cluster.
+* Ask the cluster owner or administrator to add you as a notification contact:
+ifdef::openshift-rosa[]
+xref:../rosa-cluster-notifications.adoc#cluster-notification-emails_rosa-cluster-notifications[Cluster notification emails].
+endif::openshift-rosa[]
+ifdef::openshift-dedicated[]
+xref:../osd-cluster-notifications.adoc#cluster-notification-emails_osd-cluster-notifications[Cluster notification emails].
+endif::openshift-dedicated[]
+
+.If your cluster does not receive notifications
+* Ensure that your cluster is registered to the Hybrid Cloud Console: link:https://console.redhat.com/openshift/register[Register a cluster]
```

Comment:
Not applicable to ROSA/OSD

Reply:
@jewzaam Not applicable because it happens automatically rather than that it is not necessary, right? There's no way that a customer would manage to un-register after install?

Reply:
Commented out rather than deleting, with a note that this will need to be added in conditions if ARO shares source with ROSA or OSD docs.